### DISCUSSION TITLE
Video download

### DISCUSSION THREAD

**Discussion by Habil99** [igordanchenko/yet-another-react-lightbox#268] [Q&A] [answered]
Hi.

I use yet-another-react-lightbox to display video and images and need to add download option for both of them. But download button is disabled when current slide contains video. Is there any solution ?

![image](https://github.com/igordanchenko/yet-another-react-lightbox/assets/67463980/db501a2a-d31e-4bd0-aa20-18f0a2a2ad3d)
![image](https://github.com/igordanchenko/yet-another-react-lightbox/assets/67463980/0d9d961d-8294-4acd-94c4-b1a9b5d3d7a9)

Thank you in advance!

**Comment by igordanchenko** [comment, marked as answer]
Video plugin uses `src` slide prop as a download URL by default. To enable Download button on video slides, you need to specify `download` slide prop.

https://yet-another-react-lightbox.com/plugins/download#Documentation
